packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -237,27 +237,47 @@ class OutlineButton extends StatefulWidget {
     super.debugFillProperties(properties);
-    properties.add(new ObjectFlagProperty<VoidCallback>('onPressed', onPressed, ifNull: 'disabled'));
-    properties.add(new DiagnosticsProperty<ButtonTextTheme>('textTheme', textTheme, defaultValue: null));
-    properties.add(new DiagnosticsProperty<Color>('textColor', textColor, defaultValue: null));
-    properties.add(new DiagnosticsProperty<Color>('disabledTextColor', disabledTextColor, defaultValue: null));
-    properties.add(new DiagnosticsProperty<Color>('color', color, defaultValue: null));
-    properties.add(new DiagnosticsProperty<Color>('highlightColor', highlightColor, defaultValue: null));
-    properties.add(new DiagnosticsProperty<Color>('splashColor', splashColor, defaultValue: null));
-    properties.add(new DiagnosticsProperty<double>('highlightElevation', highlightElevation, defaultValue: 2.0));
-    properties.add(new DiagnosticsProperty<BorderSide>('borderSide', borderSide, defaultValue: null));
-    properties.add(new DiagnosticsProperty<Color>('disabledBorderColor', disabledBorderColor, defaultValue: null));
-    properties.add(new DiagnosticsProperty<Color>('highlightedBorderColor', highlightedBorderColor, defaultValue: null));
-    properties.add(new DiagnosticsProperty<EdgeInsetsGeometry>('padding', padding, defaultValue: null));
-    properties.add(new DiagnosticsProperty<ShapeBorder>('shape', shape, defaultValue: null));
+    properties.add(new ObjectFlagProperty<VoidCallback>('onPressed', onPressed,
+        ifNull: 'disabled'));
+    properties.add(new DiagnosticsProperty<ButtonTextTheme>(
+        'textTheme', textTheme,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<Color>('textColor', textColor,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<Color>(
+        'disabledTextColor', disabledTextColor,
+        defaultValue: null));
+    properties.add(
+        new DiagnosticsProperty<Color>('color', color, defaultValue: null));
+    properties.add(new DiagnosticsProperty<Color>(
+        'highlightColor', highlightColor,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<Color>('splashColor', splashColor,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<double>(
+        'highlightElevation', highlightElevation,
+        defaultValue: 2.0));
+    properties.add(new DiagnosticsProperty<BorderSide>('borderSide', borderSide,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<Color>(
+        'disabledBorderColor', disabledBorderColor,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<Color>(
+        'highlightedBorderColor', highlightedBorderColor,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<EdgeInsetsGeometry>(
+        'padding', padding,
+        defaultValue: null));
+    properties.add(new DiagnosticsProperty<ShapeBorder>('shape', shape,
+        defaultValue: null));
```

Comment:
please undo the dartfmt changes that make things less readable, such as these.

You might be able to set dartfmt to a longer line limit to make things less bad.

Reply:
Done